.github/ISSUE_TEMPLATE/docs.yaml

```diff
@@ -0,0 +1,19 @@
+name: Documentation feedback
+description: Give feedback on the Grafana Alloy documentation.
+labels: ["type/docs"]
+body:
+  - type: input
+    id: page
+    attributes:
+      label: Page
+      description: The page you're providing feedback on.
+      placeholder: /docs/alloy/latest/getting-started/
```

Comment:
Does this target matter? or should it be `/docs/alloy/latest/`

Reply:
The placeholder is just the sample text to show, so I wanted to give an example of a full page to use. We could make it simpler. 

(But the intent is that this field gets pre-filled by the "Contribute to Docs" button)

Reply:
Actually, I think I'm going to change this to be the full URL. One of the issues with the existing docs issues is that you have to do a bit of work to find the page; having a full URL would be really useful here.